pandas/tests/io/json/test_pandas.py

```diff
@@ -1259,19 +1259,10 @@ def test_to_json_large_numbers(self, bigNum):
     def test_read_json_large_numbers(self, bigNum):
         # GH20599
 
-        series = Series(bigNum, dtype=object, index=["articleId"])
-        json = '{"articleId":' + str(bigNum) + "}"
-        with pytest.raises(ValueError):
-            json = StringIO(json)
-            result = read_json(json)
-            tm.assert_series_equal(series, result)
-
-        df = DataFrame(bigNum, dtype=object, index=["articleId"], columns=[0])
-        json = '{"0":{"articleId":' + str(bigNum) + "}}"
-        with pytest.raises(ValueError):
-            json = StringIO(json)
-            result = read_json(json)
-            tm.assert_frame_equal(df, result)
+        json = StringIO('{"articleId":' + str(bigNum) + "}")
+        msg = r"Value is too small|Value is too big"
+        with pytest.raises(ValueError, match=msg):
+            read_json(json)
```

Comment:
Looks like the line for testing the string `'{"0":{"articleId":' + str(bigNum) + "}}"` is lost with these changes. The two cases `'{"articleId":' + str(bigNum) + "}"` and `'{"0":{"articleId":' + str(bigNum) + "}}"` are different, so I think both should be kept.

Reply:
thanks. looks like I misread it to do the same thing twice.

Reply:
fixed